### PR TITLE
Bug/fixes per devx review

### PR DESCRIPTION
**Pull request by deniszpua**
Resolves hypertrack/issues#1569

### CODE REVIEW

File: app/src/main/java/com/hypertrack/views/quickstart/MainActivity.java [hunk 1]
```diff
@@ -30,15 +30,15 @@ protected void onResume() {
...
+        mHyperTrackView = HyperTrackViews.getInstance(this, PUBLISHABLE_KEY);
+        mHyperTrackView.getDeviceMovementStatus("A4005505-D469-481E-AD2E-123456789AB",
                 new Consumer<MovementStatus>() {
                     @Override
                     public void accept(MovementStatus movementStatus) {
                         Log.d(TAG, "Got movement status data " + movementStatus);
                     }
                 });
-        mHyperTrackData.subscribeToDeviceUpdates("A4005505-D469-481E-AD2E-123456789AB",
+        mHyperTrackView.subscribeToDeviceUpdates("A4005505-D469-481E-AD2E-123456789AB",
```
Comment (agraebe) [inline app/src/main/java/com/hypertrack/views/quickstart/MainActivity.java]: maybe we can make the device_id a property and add a comment next to pointing out it should be replaced (same for PUBLISHABLE_KEY)?

File: README.md [hunk 2]
```diff
@@ -98,3 +100,49 @@ Make sure you've stop updates, once you're done since you can end up with leaked
...
+you can build exactly the same views you really want. Anyway, ready to use UI elements will be added
+to upcoming library releases.
+
+#### Track multiple devices
+You can subscribe to more than one device by executing `subscribeToDeviceUpdates` multiple times with
+different device ids. Subscription updates consumer can be the same (e.g. map fragment), since each
+update has device id field for identification.
+
+#### Trips Tracking
+Each trip has a device id, for which it was created, pass that id to `subscribeToDeviceUpdates` and
```
Comment (agraebe) [inline README.md]: We could point out that the trips updates include trip_id to track specific trips as well

Comment (deniszpua) [reply README.md]: Unfortunately we can't really track trips yet (we don't have per-trip subscriptions). Although, it is possible to create such a logic in views sdk itself, I believe it should appear in future on backend, so I don't want to perform work that will lose its value in nearest future.

File: app/src/main/java/com/hypertrack/views/quickstart/MainActivity.java (hunk 1 above)
Comment (deniszpua) [reply app/src/main/java/com/hypertrack/views/quickstart/MainActivity.java]: Done!

**Pull request merged**